src/syntax/translation/resugar/tests.rs

```diff
@@ -10,70 +10,70 @@ fn index() -> ByteIndex {
-                    params: vec![],
-                    return_ann: None,
-                    body: concrete::Term::Universe(span(), None),
-                },
-            ],
-        };
-
-        assert_eq!(core_module.resugar(&ResugarEnv::new()), concrete_module);
-    }
-
-    // TODO: moare tests
-}
+// mod module {
+//     use super::*;
+
+//     #[test]
+//     fn shadow_keyword() {
+//         let var_else1 = FreeVar::fresh_named("else");
+//         let var_else2 = FreeVar::fresh_named("else");
+
+//         let core_module = core::Module {
+//             items: vec![
+//                 core::Item::Declaration {
+//                     label: Label("else".to_owned()),
+//                     binder: Binder(var_else1.clone()),
+//                     term: core::RcTerm::from(core::Term::universe(1)),
+//                 },
+//                 core::Item::Definition {
+//                     label: Label("else".to_owned()),
+//                     binder: Binder(var_else1.clone()),
+//                     term: core::RcTerm::from(core::Term::universe(0)),
+//                 },
+//                 // This shouldn't happen, but let's test what happens anyway!
+//                 core::Item::Declaration {
+//                     label: Label("else".to_owned()),
+//                     binder: Binder(var_else2.clone()),
+//                     term: core::RcTerm::from(core::Term::universe(1)),
+//                 },
+//                 core::Item::Definition {
+//                     label: Label("else".to_owned()),
+//                     binder: Binder(var_else2.clone()),
+//                     term: core::RcTerm::from(core::Term::universe(0)),
+//                 },
+//             ],
+//         };
+
+//         let concrete_module = concrete::Module::Valid {
+//             items: vec![
+//                 concrete::Item::Declaration {
+//                     name: (index(), "else1".to_owned()),
+//                     ann: concrete::Term::Universe(span(), Some(1)),
+//                 },
+//                 concrete::Item::Definition {
+//                     name: (index(), "else1".to_owned()),
+//                     params: vec![],
+//                     return_ann: None,
+//                     body: concrete::Term::Universe(span(), None),
+//                 },
+//                 concrete::Item::Declaration {
+//                     name: (index(), "else2".to_owned()),
+//                     ann: concrete::Term::Universe(span(), Some(1)),
+//                 },
+//                 concrete::Item::Definition {
+//                     name: (index(), "else2".to_owned()),
+//                     params: vec![],
+//                     return_ann: None,
+//                     body: concrete::Term::Universe(span(), None),
+//                 },
+//             ],
+//         };
+
+//         assert_eq!(core_module.resugar(&ResugarEnv::new()), concrete_module);
+//     }
+
+//     // TODO: moare tests
+// }
```

Comment:
Gotta test this using let expressions, and perhaps move them to the `desugar` tests